Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.6-stretch
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LC_ALL=C.UTF-8
+ENV DJANGO_SETTINGS_MODULE=production_settings
+
+ENV POSTGRES_DB=pretix
+ENV POSTGRES_USER=pretix
+ENV POSTGRES_PASSWORD=pretix
+
+ENV DJANGO_SECRET=eehucheethie1cuW7bieKooviecoFooj9yepai1Oec3aevah3f
+ENV MAIL_FROM=tickets@ChangeMeToYourPretixDomain.arpa
+ENV INSTANCE_NAME="My pretix installation"
+ENV CURRENCY=EUR
+
+RUN apt-get update \
+    && apt-get install -y git libxml2-dev libxslt1-dev python-dev python-virtualenv locales \
+        libffi-dev build-essential python3-dev zlib1g-dev libssl-dev gettext libpq-dev \
+        default-libmysqlclient-dev libjpeg-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN dpkg-reconfigure locales \
+	&& locale-gen C.UTF-8 \
+	&& /usr/sbin/update-locale LANG=C.UTF-8
+RUN mkdir /etc/pretix /data /static
+RUN pip3 install -U pip wheel setuptools uwsgi
+
```

Comment:
I think you could use the same RUN command for lines 16, 23, 26 and 27
and you should reduce the docker container size

Reply:
That will make the build cache unusable.
And this would also not reduce the size, as nothing gets deleted, but added here.